.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+      tabWidth: 6,
```

Comment:
why have we increased tabWidth
Ideal value is 2 
https://developers.google.com/style/html-formatting

Reply:
with 6 tabs the code is more readable.

Reply:
two major drawbacks
1. universally accepted line length is 80 chars (because post that line cannot be viewed in on go), so with 6 space indentation even if a component is 4 level down, then 24 space will be added 
i.e. 30% wasted
2. As 6 is not the industry standard, new developer will face difficulty working here